app/javascript/app/ssn-field.js

```diff
@@ -48,8 +48,20 @@ function formatSSNField() {
 
       sync();
       toggle.addEventListener('change', sync);
+
+      const ssnBox = document.getElementById('doc_auth_ssn');
```

Comment:
This logic is called for each matching SSN on a page. If there were multiple here, I expect it could cause conflicts. In any case, would I be correct in assuming `ssnBox` is just the scoped `input` variable?

```suggestion
      const ssnBox = input;
```

(or better yet, avoid `ssnBox` altogether and reference `input` directly)

Reply:
A specific id should occur only once on a page.  An input box can occur multiple times.

Reply:
What I mean is that this code is within a loop:

```js
[].slice.call(inputs).forEach((input, i) => {
  // ...
  const ssnBox = document.getElementById('doc_auth_ssn');
});
```

But I assume we only want this run at most once per SSN field. So we'd either want this moved outside the `forEach`, or (preferably, and largely already implemented in recent changes) avoiding the ID'd field reference and just use the `input` variable.

Reply:
ah I see.